Project.toml

```
@@ -61,7 +63,7 @@ EllipticalSliceSampling = "0.5, 1, 2"
 ForwardDiff = "0.10.3"
 Libtask = "0.7, 0.8"
 LogDensityProblems = "2"
-LogDensityProblemsAD = "1.4"
+LogDensityProblemsAD = "1.6"
```

Version 1.6.0 was broken:
```suggestion
LogDensityProblemsAD = "1.6.1"
```